src/daily-auths-report.tsx

```diff
@@ -256,16 +262,25 @@ function DailyAuthsReport(): VNode {
     setAllAgencies(allAgencies);
   }, [data]);
 
+  useEffect(() => {
+    const listener = () => setWidth(ref.current?.offsetWidth);
+
+    listener();
+
+    window.addEventListener("resize", listener);
+    return () => window.removeEventListener("resize", listener);
+  });
+
   return (
-    <div>
+    <div ref={ref}>
       <PlotComponent
```

Comment:
I hadn't noticed 'til now, but it'd be nice to split this file up. Typically I'd only expect one component per file, so pulling out `Plot` to its own file might be a nice way to handle that?

Reply:
I knew you'd comment on that! 😝 My thinking was more along the lines of "if we aren't going to reference it from a second file, it's fine to keep here for now", but I will move PlotComponent out

Reply:
88e2fa3a29aba7fb6011aedcb68a8f33ae17f3d3